CombineExamples.playground/Pages/Publishers and Subscribers.xcplaygroundpage/Contents.swift

```diff
@@ -21,16 +21,50 @@ publisher.send("Hello, old friend")
 
 let textObserver: PassthroughSubject<String, Never> = .init()
 
-let textCancellable = textObserver
+
+var textCancellable = textObserver
     .sink { text in
         print(text)
 }
-
 textObserver.send("H")
 textObserver.send("e")
 textObserver.send("l")
 textObserver.send("l")
 textObserver.send("o")
 
+/// cancel our current subscription
+textCancellable.cancel()
+
 /// When you subscribe to a publisher you get a token back, a `Cancellable`. The subscription lives until the Cancellable is deallocated or `.cancel()` is called on it, which means you need to store the cancellable somewhere to get any output. If the cancellable is deallocated, your subscription is terminated
 
+
+func subscribeTo(_ stream: PassthroughSubject<String, Never>) {
+    // since this `localCancellable` is in this functions scope the subscrition is cancelled directly after the function extits
```

Comment:
```Swift
// since this `localCancellable` is in this [functions -> function's] scope the [subscrition -> subscription] is cancelled directly after the function [extits -> exits]
```